dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -170,6 +174,24 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 
 	const { absoluteServerTimes = false } = front.config.switches;
 
+	const fallBackAspectRatio = (collectionType: DCRContainerType) => {
```

Comment:
This is nitpicking of the highest order, but fallback is one word so this would be `fallbackAspectRatio`. (But totally unimportant so feel free to leave as is 😅)